### ISSUE TITLE
Adafruit library download

### ISSUE THREAD

**Issue by jayjabour** [adafruit/Adafruit_Blinka#264]
I am trying to install Adafruit motorkit library for their Raspberry Pi Motor Controller hat.  I have Python 3.7.0 installed, but the download process will not install with the reason given 'Python 3.5 or later is required'.  Python 3.5 is on the Pi, but I am using new version 3.7.  Any help appreciated.

**Comment by makermelissa** [comment]
Try explicitly typing python3 or pip3. It's possible that you also have python 2 (both can exist simultaneously) and that's what it is running with. What is the command you're typing to run it?

**Comment by jayjabour** [comment]
Melissa

Thanks for the reply.  The command I used was;

        sudo pip3 install adafruit-circuitpython-motorkit

I am a Linux novice, a lot of experience with C++ and Java on PC and Mac,
but I have avoided Linux.  Thanks for any help you can give.

Jay Jabour

**Comment by makermelissa** [comment]
Could you explain more what you meant by:
> Python 3.5 is on the Pi, but I am using new version 3.7.

Meaning you are able to run either?
Also, are you running Raspbian or a different OS? We officially support Raspbian, but I've heard people have been able to get it working on Ubuntu and Diet Pi.

**Comment by jayjabour** [comment]
Melissa

The Raspberry Pi came with Python 3.5.  When I tried to install the Motor
driver support it gave me a message that the library does not work with
Python version 3.5.  I then updated the Python version to 3.7.  Again, tried
to install the Motor driver library and I still got the message that the
library does not support versions before 3.5.  I can now run either 3.5 and
3.7.

I am using Raspberry Pi Raspbian version of Linux.

William Jabour
